src/App.js

```diff
@@ -63,6 +72,31 @@ class App extends React.Component {
                 <Switch>                             
                     <Route exact path='/'>
                         <HomePage {...state} updateData={updateData} />                                  
+                    </Route>
+                    <Route path='/login'>
+                        <LoginPage {
+                            ...{
+                                buttonText: 'Login',
+                                linkText: 'Logout?',
+                                linkUrl: '/register',
+                                onSubmitLoginForm: this.handlerSubmitLoginForm,
+                                onChangeLoginInput: this.handlerChangeLoginInput
+                            }
```

Comment:
looks odd, why do you need spread here? just use props

Reply:
> looks odd, why do you need spread here? just use props

should I create fields in the status of the App component: email and password?